### PR TITLE
Label aspect-ratio tests as interop-2021

### PR DESCRIPTION
**Pull request by foolip**
Based on https://raw.githubusercontent.com/Ecosystem-Infra/wpt-results-analysis/main/compat-2021/aspect-ratio-tests.txt.

### CODE REVIEW

File: css/css-sizing/aspect-ratio/META.yml [hunk 1]
```diff
@@ -1,4 +1,159 @@
...
+  - test: intrinsic-size-003.html
+  - test: intrinsic-size-004.html
+  - test: intrinsic-size-005.html
+  - test: intrinsic-size-006.html
+  - test: intrinsic-size-007.html
+  - test: intrinsic-size-008.html
+  - test: intrinsic-size-009.html
+  - test: intrinsic-size-010.html
+  - test: intrinsic-size-011.html
+  - test: parsing/aspect-ratio-computed.html
```
Comment (foolip) [inline css/css-sizing/aspect-ratio/META.yml:103]: Good, this hit https://github.com/web-platform-tests/wpt-metadata/blob/0064a416666d3b0e1194f9ba3bbd5f9d7efbb2b2/go_test/parse_test.go#L57. Fixing...

**Comment by KyleJu** [comment]
Checks should pass now with #2201. Please rerun this check after merge #2201.

**Comment by foolip** [comment]
Thanks @KyleJu, this is merged now. Trying it out in staging:
https://staging.wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021

I see "Failed to fetch test runs", could this be a memory issue?

**Comment by KyleJu** [comment]
> Thanks @KyleJu, this is merged now. Trying it out in staging: https://staging.wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021
> 
> I see "Failed to fetch test runs", could this be a memory issue?

Yup it is a memory issue. Here is a workaround by showing less runs https://staging.wpt.fyi/results/?q=label%3Ainterop-2021&run_id=6739214003601408

**Comment by foolip** [comment]
@KyleJu it looks a lot like only the first file from each META.yml file has been correctly labeled in the search cache. Did I use the wrong syntax in this PR, or what do you think is wrong?

**Comment by KyleJu** [comment]
> @KyleJu it looks a lot like only the first file from each META.yml file has been correctly labeled in the search cache. Did I use the wrong syntax in this PR, or what do you think is wrong?

Thanks for the report :) Sent out a fix https://github.com/web-platform-tests/wpt.fyi/pull/2695

**Pull request merged**